templates/zerver/integrations/capistrano.md

```diff
@@ -0,0 +1,34 @@
+First, download and install our [Python bindings and example scripts](/api) on your Capistrano server. Once you've
```

Comment:
I think this line can be made part of a macro. Asking users to download our bindings is something a lot of these integrations do. :)